### PR TITLE
Code Review - Initial Release

### PR DESCRIPTION
**Pull request by xhlulu**
lorem ipsum something foo bar

### CODE REVIEW

**Comment by matthewchan15** [comment]
Sample Comment

File: app.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import pandas as pd
+import numpy as np
+import plotly.graph_objs as go
+from dash.dependencies import Input, Output, State
+
+app = dash.Dash(__name__)
+server = app.server
+
+# Custom Script for Heroku
+if 'DYNO' in os.environ:
```
Comment (shammamah) [inline app.py]: Sample comment for code review.

File: app.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        'external_url': 'https://cdn.rawgit.com/chriddyp/ca0d8f02a1659981a0ea7f013a378bbd/raw/e79f3f789517deec58f41251f7dbb6bee72c44ab/plotly_ga.js'
+    })
+
+
+app.layout = html.Div([
+    # Banner display
+    html.Div([
+        html.H2(
+            'App Name',
+            id='title'
```
Comment (shammamah) [inline app.py]: Another sample comment from the code review.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,12 @@
+*.zip
+*.iml
+*.xml
+mnist2500_labels.txt
```
Comment (matthewchan15) [inline .gitignore]: Sample

File: app.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import os
+
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import pandas as pd
+import numpy as np
+import plotly.graph_objs as go
```
Comment (matthewchan15) [inline app.py]: Sample

File: custom_styles.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+
+@keyframes fadein {
+    from { opacity: 0; }
+    to   { opacity: 1; }
+}
+
+/* Firefox < 16 */
+@-moz-keyframes fadein {
```
Comment (matthewchan15) [inline custom_styles.css]: Sample

File: requirements.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
+dash==0.21.1
+dash-renderer==0.12.1
+dash-html-components==0.10.1
+dash-core-components==0.22.1
+numpy==1.14.3
+pandas==0.20.3
+plotly==2.5.1
+scikit-learn==0.19.1
```
Comment (matthewchan15) [inline requirements.txt]: Sample

File: .gitignore (hunk 3 above)
Comment (xhlulu) [reply .gitignore]: No

File: app.py (hunk 4 above)
Comment (xhlulu) [reply app.py]: Yes

File: custom_styles.css (hunk 5 above)
Comment (xhlulu) [reply custom_styles.css]: No

File: requirements.txt (hunk 6 above)
Comment (xhlulu) [reply requirements.txt]: NO

File: app.py (hunk 1 above)
Comment (xhlulu) [reply app.py]: yes

File: app.py (hunk 2 above)
Comment (xhlulu) [reply app.py]: yes

**Comment by matthewchan15** [comment]
👯

**Comment by matthewchan15** [comment]
💃

**Pull request merged**